.github/workflows/main.yml

```
@@ -0,0 +1,24 @@
+name: main
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

```suggestion
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```
you can simplify this, here's an example https://github.com/xconnio/wampproto.py/blob/main/.github/workflows/main.yaml